Dockerfile

```diff
@@ -32,4 +49,5 @@ COPY NAMESPACE /tmp/install/repgen/NAMESPACE
 RUN cd /tmp/install/ && R CMD build repgen && \
   mv /tmp/install/repgen_*.tar.gz /tmp/install/repgen.tar.gz && \
   Rscript -e "install.packages(\"/tmp/install/repgen.tar.gz\", repos = NULL, type=\"source\")" && \
+  ls /opt/rserve/R_libs | if [ $(grep -c repgen) -eq 0 ]; then echo "Repgen build failed" && exit 1; fi && \
```

Comment:
Previously the build would still continue if the Repgen build/install failed because Rscript always returns exit code 0, even if the script it ran exited with an error. This line checks that the package was _actually_ installed before continuing.